ComponentsInitialization.js

```diff
@@ -16,27 +20,20 @@ define(function (require) {
 
                     <div id="footer">
                         <div id="footerHeader">
-                            <div id="consoleButtonContainer">
-                                <ul className="btn nav nav-tabs" role="tablist" id="tabButton">
-                                    <li role="presentation" className="active" id="consoleButton"><a href="#console" aria-controls="console" role="tab" data-toggle="tab"><i className="fa fa-terminal"></i> Console</a></li>
-                                    <li role="presentation" id="pythonConsoleButton" style={{ display: 'none' }}><a href="#pythonConsole" aria-controls="pythonConsole" role="tab" data-toggle="tab"><i className="fa fa-terminal"></i> Python</a></li>
-                                </ul>
-                                <div className='netpyneversion'>NetPyNE UI Alpha version 0.4</div>
-                                <div className="tab-content">
-                                    <div role="tabpanel" className="tab-pane active" id="console">Console Loading...</div>
-                                    <div role="tabpanel" id="pythonConsole" className="tab-pane  panel panel-default"></div>
-                                </div>
-                            </div>
+                            <TabbedDrawer labels={["Console", "Python"]} iconClass={["fa fa-terminal", "fa fa-flask"]} >
+                                <Console />
+                                <PythonConsole pythonNotebookPath={pythonNotebookPath} />
+                            </TabbedDrawer>
                         </div>
                     </div>
                 </div>
             );
         }
         ReactDOM.render(<App />, document.querySelector('#mainContainer'));
 
-        GEPPETTO.ComponentFactory.addComponent('CONSOLE', {}, document.getElementById("console"));
-        var pythonNotebookPath = "http://" + window.location.hostname + ":" + window.location.port + "/notebooks/notebook.ipynb";
-        GEPPETTO.ComponentFactory.addComponent('PYTHONCONSOLE', { pythonNotebookPath: pythonNotebookPath }, document.getElementById("pythonConsole"));
+        // Old declaration for TabbedDrawer plus children using the component factory.
+        // Children's props have to be declared into the array childrenProps
+        //GEPPETTO.ComponentFactory.addComponent('DRAWER', {children: [Console, PythonConsole], childrenProps: [null, { pythonNotebookPath: pythonNotebookPath }], labels: ["Console", "Python"], iconClass: ["fa fa-terminal", "fa fa-flask"]}, document.getElementById("footerHeader"));
```

Comment:
I guess you keep this bit as a comment to ease the PR review. Unless there is another reason for this I am missing I would remove these lines before merging...

Reply:
That was it :), cleaning it now.